core/src/ModelMetadata.cpp

```diff
@@ -233,6 +233,10 @@ void ModelMetadata::incrementTime(const Duration& step)
 #endif
 }
 
+void ModelMetadata::setInitialFileName(const std::string filename) { this->initFile = filename; }
+void ModelMetadata::setFinalFileName(const std::string filename) { this->finalFile = filename; }
+void ModelMetadata::setRestartPeriod(const Duration period) { this->period = period; }
+
```

Comment:
If the getter and setters both exist and don't do anything special, why not just make these public variables?

Reply:
This would be simpler, indeed. I decided to fix it here rather than following up (see f3fba49a). Will merge once tests pass.